docs/setup/new_environment.md

```diff
@@ -289,3 +289,9 @@ $ commcare-cloud monolith django-manage make_superuser {email}
 ```
 
 where `{email}` is the email address you would like to use as the username.
+
+## Step 13: [Optional] Add a new build
+
+To make a build of an application in CommCare, you will need to add a CommCare build to your server.
```

Comment:
This is a very confusing sentence... `to make a build... add a build`. 

```suggestion
In order to create new versions of applications created in the CommCareHQ app builder, you will need to add the latest `CommCare Mobile` and `CommCare Core` builds to your server.
```

Reply:
Agreed. Sometimes it feels like we could replicate https://en.wikipedia.org/wiki/Buffalo_buffalo_Buffalo_buffalo_buffalo_buffalo_Buffalo_buffalo with our usages of apps and builds